scripts/publish.js

```diff
@@ -112,31 +118,27 @@ function copyReadme() {
   return fs.copy('readme.md', `${distTarget}/readme.md`);
 }
 
-function copyScssFiles() {
-  console.log('Copying SCSS files...');
-  const onlyScssFiles = (input) => ['', '.scss'].includes(path.extname(input));
-  return fs.copy(`${coreLibDir}/scss`, `${distTarget}/scss`, { filter: onlyScssFiles });
-}
-
-function copyScssUtilsFile() {
-  console.log('Copying SCSS utils file...');
-  const onlyScssFiles = (input) => ['', '.scss'].includes(path.extname(input));
-  return fs.copy(`${angularLibDir}/scss/_utils.scss`, `${distTarget}/scss/_utils.scss`, {
-    filter: onlyScssFiles,
+function forwardCoreScssFilesToDesignsystem() {
+  console.log('Forwarding core SCSS files to designsystem ...');
+  return new Promise((resolve) => {
+    forwardScssFiles(`${coreLibDir}/scss`, `${distTarget}/scss`, packageAlias, libsRootDir);
```

Comment:
Optional.

Just a thought: Would it be more "future proof" to

* rename this function to `forwardCoreScssFiles()` and
* pass the destination as argument

or would that be a premature optimization?

Reply:
I'm not particularly a fan of passing in arguments to the function as it isn't done for any of the other functions in the ".then" chain at the bottom `publish..js`. 

How about renaming the step to `forwardCoreScssFiles` but having it take no arguments. Instead we will just have a call to `forwardScssFiles` for each library we want the files forwarded to. 

The `forwardCoreScssFiles` can then look something like: 
```
function forwardCoreScssFilesToDesignsystem() {
  console.log('Forwarding core SCSS files ...');
  const locations = [{from: 'sourcePath', to: 'targetPath'}, {from: 'sourcePath2', to: 'targetPath2'}]
  return new Promise((resolve) => {
     locations.forEach(({sourcePath, targetPath}) => {...}
   }
...
``

Reply:
Ended up doing the following: 
```javascript
function forwardCoreScssFiles() {
  console.log('Forwarding core SCSS files...');

  return new Promise((resolve) => {
    pathsToForwardCoreScssFilesTo.forEach((targetPath) => {
      forwardScssFiles(`${coreLibDir}/scss`, targetPath, packageAlias, libsRootDir);
      resolve();
    });
  });
}
```

`pathsToForwardCoreScssFilesTo` can be found at the top of the file - closer to the variables it references.